canonical-web-and-design/maas-ui
# Display a delete confirmation on user details

Done:
- Show a delete confirmation when deleting user from a user details page. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1355

QA:
- View the list of users and click on a user.
- Click the delete button.
- A confirmation should appear.
- Click cancel, it should disappear.
